liquidator/liquidator.js

```diff
@@ -174,10 +241,13 @@ class Liquidator {
         this.logger.error({
           at: "Liquidator",
           message:
-            "Cannot liquidate position: not enough synthetic (or large enough approval) to initiate liquidation✋",
+            "Failed to liquidate position: not enough synthetic (or large enough approval) to initiate liquidation❌",
```

Comment:
Changed this language to signal that the bot had enough permission by `maxTokensToLiquidateWei` to send a liquidation, but the liquidation failed for some on-chain reason

Reply:
I didn't even think of this. Good call.